front-end/src/components/Post/GovernanceSideBar/EndorseTip.tsx

```diff
@@ -44,11 +44,13 @@ const EndorseTip = ({
 	const [isCouncil, setIsCouncil] = useState(false);
 	const [forceEndorse, setForceEndorse] = useState(false);
 	const councilQueryresult = useGetCouncilMembersQuery();
-	const currentCouncil: string[] = [];
+	const [currentCouncil, setCurrentCouncil] = useState<string[]>([]);
 	const { api, isApiReady } = useContext(ApiPromiseContext);
 	const { addresses } = useContext(UserDetailsContext);
 
-	councilQueryresult.data?.councils?.[0]?.members?.forEach( member => {currentCouncil.push(member?.address);});
+	councilQueryresult.data?.councils?.[0]?.members?.forEach( member => {
+		setCurrentCouncil([...currentCouncil, member?.address]);
```

Comment:
A `if` could be needed here, I don't see why `member` would be undefined, but adding something like the following wouln't hurt:
```suggestion
		if (member)
		  setCurrentCouncil([...currentCouncil, member.address]);
		}
```